### PR TITLE
Add status comments to Github PR

### LINKED ISSUE #47: report release status on the PR
release bot should ideally post status on the PR whether the release was successful or not

logs should not be present since they may contain sensitive information

### PR DESCRIPTION
**Pull request by jpopelka**
Fixes #47

### CODE REVIEW

File: release_bot/github.py [hunk 1]
```diff
@@ -126,7 +145,9 @@ def make_new_release(self, new_release, previous_pypi_release):
                                    f"new release on github:\n{response.text}"))
                 exit(1)
         else:
-            self.logger.info(f"Released {new_release['version']} on Github")
+            msg = f"Released {new_release['version']} on Github"
```
Comment (TomasTomecek) [inline release_bot/github.py]: nit: `I just released version {new_release['version']} on Github.`

File: release_bot/releasebot.py [hunk 2]
```diff
@@ -68,13 +69,19 @@ def make_new_pypi_release(self):
...
+        self.logger.info(msg)
+        self.github.add_comment(self.new_release['pr_id'], msg)
         return True
 
     def make_new_fedora_release(self):
         if self.new_release['fedora']:
             self.logger.info("Triggering Fedora release")
             self.fedora.release(self.new_release)
             self.new_release['tempdir'].cleanup()
+            msg = "Finished Fedora release"
```
Comment (TomasTomecek) [inline release_bot/releasebot.py]: Could we also get Fedora version here?

File: release_bot/releasebot.py [hunk 3]
```diff
@@ -68,13 +69,19 @@ def make_new_pypi_release(self):
                                                                 'release-conf.yaml'))
         self.new_release.update(release_conf)
         self.pypi.release(self.new_release)
+        msg = f"Released {self.new_release['version']} on PyPI"
```
Comment (TomasTomecek) [inline release_bot/releasebot.py]: Similar as above.

File: release_bot/releasebot.py (hunk 2 above)
Comment (jpopelka) [reply release_bot/releasebot.py]: Just the branches which we successfully updated / built from.
To get exact N-V-R I'd probably need to parse `fedpkg build` output.

**Comment by jpopelka** [comment]
PTAL Tome

**Comment by TomasTomecek** [comment]
Awesome changes, this is rocking!

**Pull request merged**